index.md

```diff
@@ -16,31 +16,17 @@ Jump to:
 
 ## Summary
 
-We will add the capability of inferring with a Mixture of Experts model (a transformer with MoE, for MLP layers only) to the FlexFlow Serve framework. Once we are able to infer with an MoE model in FlexFlow, we will benchmark its per-token latency and throughput on a node with four V100 GPUs against a standard, non-accelerated service setting.
+We will add support for Mixture of Experts (MoE) Transformer models to the FlexFlow Serve framework. Once we are able to successfully run inference with the model on FlexFlow, we will benchmark its per-token latency and throughput on a node with four V100 GPUs against a standard, non-accelerated service setting.
```

Comment:
I would specify it's MoE for MLP layers only. MoE is possible for other layers

Reply:
added sentence afterwards